doc/api-extensions.md

```diff
@@ -2312,10 +2312,6 @@ The new configuration keys are `network.ovn.ca_cert`, `network.ovn.client_cert`
 ## `init_preseed_storage_volumes`
 This API extension provides the ability to configure storage volumes in preseed init.
 
-## `metrics_instances_count`
```

Comment:
Can you use a git revert of the original combined api extension commit for this please

Reply:
fixed